prac_10/wiki.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404 - Practical 10
+Wikipedia Python API
+"""
+
+import wikipedia
+from wikipedia import DisambiguationError, PageError
+
+query = input("Enter page title or query phrase in quotation marks: ")
+while query != "":
+    try:
+        search = wikipedia.page(query)
+        print(f"Title: {search.title}")
+        print(f"URL: {search.url}")
+        print(f"Summary: {wikipedia.summary(str(query))}")
```

Comment:
Hi Trijeet, well done with this prac.

Some feedback: To extend your wiki you could prompt the user for another search after a valid search, for example, what if a user wants to search for something else. Right now you only prompt the user again when an invalid search is entered, would be easy to implement :)

Reply:
Thank you @lador1 for the feedback. Will work on the wiki file to improve it.